packages/components/accordion/package.json

```diff
@@ -22,6 +22,9 @@
     "ignore:rm": "git clean -Xdf",
     "generate": "stencil generate",
     "doc:api": "typedoc",
+    "doc": "yarn run doc:accordion && yarn run doc:accordion-group && node ../../../scripts/generate-typedoc-md.js --multiple",
+    "doc:accordion": "typedoc src/components/osds-accordion/public-api.ts --out ./docs-api/accordion --json ./docs-api/accordion/typedoc.json",
+    "doc:accordion-group": "typedoc src/components/osds-accordion-group/public-api.ts --out ./docs-api/accordion-group --json ./docs-api/accordion-group/typedoc.json",
```

Comment:
Food for thought, maybe those (and radio) should be separated into different components